ui/app/components/ui/readonly-input.js

```diff
@@ -32,3 +25,10 @@ ReadOnlyInput.prototype.render = function ReadOnlyInput () {
   )
 }
 
+ReadOnlyInput.propTypes = {
+  wrapperClass: PropTypes.string,
+  inputClass: PropTypes.string,
+  value: PropTypes.string,
+  textarea: PropTypes.bool.isRequired,
```

Comment:
```suggestion
  textarea: PropTypes.bool,
```
This should not be required; it's not given in either of the two places this component is used.